File: BUSQUEDA LINEAL/BUSQUEDALINEAL.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    vector<int> arreglo = {10, 20, 30, 40, 50};
    int num;
    bool encontrado = false;

    cout << "¿Qué número buscas? ";
    cin >> num;

    for (int elemento : arreglo) {
        if (elemento == num) {
            encontrado = true;
            break;
        }
    }

    if (encontrado) {
        cout << "Elemento " << num << " encontrado en el arreglo." << endl;
    } else {
        cout << "Elemento no encontrado." << endl;
    }

    return 0;
}

File: BUSQUEDA LINEAL/BUSQUEDALINEAL.js
let arreglo = [10, 20, 30, 40, 50];


let num = 30;  

let encontrado = false;
for (let elemento of arreglo) {
    if (elemento === num) {
        encontrado = true;
        break;
    }
}

if (encontrado) {
    console.log(`Elemento ${num} encontrado en el arreglo.`);
} else {
    console.log("Elemento no encontrado.");
}
